src/penn_chime/cli.py

```diff
@@ -74,7 +74,7 @@ def parse_args():
             "Doubling time before social distancing (days)",
             True,
         ),
-        ("--hospitalized-los", int, 0, None, "Hospitalized Length of Stay (days)", True),
+        ("--hospitalized-days", int, 0, None, "Hospital Length of Stay (days)", True),
```

Comment:
Probably this should read something like `Average Hospital Length of Stay` just to be clear (or `Mean`?)